example/gardener-local/gardenlet/values-provider-extensions.yaml

```diff
@@ -0,0 +1,106 @@
+config:
+  gardenClientConnection:
+    kubeconfigSecret:
+      name: gardenlet-kubeconfig
+      namespace: garden
+    bootstrapKubeconfig:
+      name: gardenlet-kubeconfig-bootstrap
+      namespace: garden
+      kubeconfig: |
+        apiVersion: v1
+        kind: Config
+        current-context: gardenlet-bootstrap
+        clusters:
+        - cluster:
+            insecure-skip-tls-verify: true
+            server: https://gardener-apiserver.relay.svc.cluster.local
+          name: default
+        contexts:
+        - context:
+            cluster: default
+            user: gardenlet-bootstrap
+          name: gardenlet-bootstrap
+        users:
+        - name: gardenlet-bootstrap
+          user:
+            token: 07401d.f395accd246ae52d
+  controllers:
+    shoot:
+      reconcileInMaintenanceOnly: true
+  featureGates:
+    HVPA: true
+    HVPAForShootedSeed: true
+    ManagedIstio: true
+    APIServerSNI: true
+    ReversedVPN: true
+    CopyEtcdBackupsDuringControlPlaneMigration: true
+    DefaultSeccompProfile: true
+    CoreDNSQueryRewriting: true
+  logging:
+    enabled: true
+    loki:
+      enabled: true
+    shootNodeLogging:
+      shootPurposes:
+      - "infrastructure"
+      - "production"
+      - "development"
+      - "evaluation"
```

Comment:
Do we have to duplicate this here? Can't we rely on the `values.yaml` like for the HA seeds?

Reply:
Ok, now `values-provider-extensions.yaml` is added to the deploy profile in skaffold instead of replacing the standard file like for the HA seeds.
Additionally, following @timuthy proposal users could write their custom values into `example/provider-extensions/gardenlet/values.yaml`